vnext/Microsoft.ReactNative/ReactHost/React.h

```diff
@@ -305,6 +305,14 @@ struct ReactOptions {
       bool value) noexcept;
   static bool UseDirectDebugger(winrt::Microsoft::ReactNative::IReactPropertyBag const &properties) noexcept;
 
+  //! "The URL that is provided by the Linking.getInitialURL() call."
+  void SetInitialUrl(winrt::hstring const &url) noexcept;
```

Comment:
ReactOptions feels like a weird place to put this, perhaps we should create a "helper" class for the InitialUrlProperty() on ReactPropertyBag and go straight from ReactInstanceSettings to ReactPropertyBag...

Reply:
Yeah, generally we want to keep ReactOptions, and ReactInstanceSettings free of specific native module logic.  We want to keep those classes free of specific platform / implementation details.  -- We share the same interface with the office platform, but there have been numerous other react-native platforms that have shared this code, and they wouldn't all have a linking native module.

I'd prefer having a `Linking` class that you use to configure the property, without having to pollute the core interfaces. This also provides more examples of how we expect community modules to be able to configure themselves without modifying the core classes.